src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -30,15 +30,15 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
             fontSize: 12,
             color: COLORS.signed_primary,
             fontWeight: '500',
-            paddingHorizontal: 8
+            paddingHorizontal: 8,
+            lineHeight: 22
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The lineHeight property has been added to the Text style. This will affect the spacing between lines of text within this component. Ensure that this change does not negatively impact the readability or layout of the text.
```diff
+            lineHeight: 22
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->